app/code/Magento/Newsletter/Test/Mftf/Test/CustomTestNewsletter.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CustomTestNewsletter">
+        <annotations>
+            <features value="Newsletter"/>
+            <stories value="Newsletter subscription"/>
+            <testCaseId value="TASK-1"/>
+            <useCaseId value="USECASE-1"/>
+            <group value="Newsletter"/>
```

Comment:
Group value should start from small letter
Also add one more group
`<group value="customer"/>`